frontend/src/ui/components/EmailActivationAlertMessage/styles.scss

```diff
@@ -0,0 +1,26 @@
+@import '~styles/theme';
+
+
+.email-activation-alert {
+  background-color: #fff3cd;
+  color: #856404;
+  padding: 15px;
+  margin: 0 -15px;
+
+  display: flex;
+  align-items: center;
```

Comment:
@giovannicimolin a small nit pick, we don't need to explicitly mention `display: flex` and `align-items: center` as we are using bootstrap classes in our `div` container.
Also, as @xitij2000 pointed out we can put the hex in the `themes.scss` :)